contrib/ohi-release-notes/action.yml

```diff
@@ -14,6 +14,15 @@ inputs:
     description: Sets the link dependency dictionary.
     default: ''
 outputs:
+  is-empty:
+    description: "Outputs if there is no new release to be done"
+    value: ${{ steps.empty.outputs.is-empty }}
+  is-held:
+    description: "Outputs if changelog is held"
```

Comment:
If we put "no new release to be done" I would put here "Outputs if release is held"

Reply:
Now that you mention it, I think it should be better to change the other one to something like "Outputs if changelog is empty", wdyt?

Reply:
Done in 04fd0e8a127f94039739f1ceda6cc7fa6d332415.